source/fundamentals/crud/write-operations/change-a-document.txt

```diff
@@ -298,6 +300,9 @@ guides:
 - :doc:`Compound Operations </fundamentals/crud/compound-operations>`
 - :manual:`Update Operators </reference/operator/update/#update-operators>`
 
+For information on updating array elements, see the :doc:`Update Arrays
```

Comment:
consider changing this from a `doc` link to a `ref` link. If the title of / path to the linked page ever changes you won't have to make changes to this link if you do this.